PauloJobson/Desafio__00/style.css

```diff
@@ -0,0 +1,106 @@
+body{
+  margin: 0;
+  padding: 0;
+  align-items: center;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+#background{
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  min-width: 100%;
+  min-height: 100%;
+}
+.nav .content{
+  background-color: white;
+  height: 60px;
+  width: 100vw;
+  text-transform: uppercase;
+  text-align:right ;
+}
+
+input{
+  margin-left: 10px;
+  width: 50vw;
+  height: 5vh;
+  border-radius: 35px;
+  padding-bottom: 5px;
+  margin-bottom: 20px;
+}
+.banner {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+
+.banner video {
+    position: fixed;
+    top: 0;
+    left: 0;
+    object-fit: cover;
+    width: 100%;
+}
+
+.banner .content {
+  flex-direction: column;
+  display: flex;
+  justify-content: flex-start ;
+  z-index: 10;
+  max-width: 600px;
+ 
+}
+
+.banner .content h1 {
+  width: 50vw;
```

Comment:
Também não precisa determinar uma largura p texto, só quando realmente for necessário, quando quiser causar a quebra do texto, por exemplo.